pkg/beacon/beacon.go

```diff
@@ -23,21 +25,25 @@ var logger = log.Logger("keep-beacon")
 // internal random beacon implementation. Returns an error if this failed,
 // otherwise enters a blocked loop.
 func Initialize(
+	ctx context.Context,
 	operatorPublicKey *operator.PublicKey,
-	beaconChain beaconchain.Interface,
+	// TODO: Get rid of legacy Ethereum chain
+	legacyBeaconChain beaconchain.Interface,
```

Comment:
s/`legacyBeaconChain`/`beaconChainV1` ? Exactly same reasons as in `legacyChainProvider`-related comment.

Reply:
https://github.com/keep-network/keep-core/pull/3077/commits/dc2cad879bebd36c3c4983ae2493b5e02816ccf8